master/master.cfg

```diff
@@ -311,7 +309,11 @@ for name, worker_name, buildfactory, stability, tier in BUILDERS:
     if any(pattern in buildername for pattern in NO_NOTIFICATION):
         continue
 
-    pull_request_builders.append(buildername)
+    if stability == STABLE:
+        stable_pull_request_builders.append(buildername)
+    else:
+        unstable_pull_request_builders.append(buildername)
```

Comment:
You should move this before NO_NOTIFICATION test.

Reply:
done